libraries/AP_Landing/AP_Landing.cpp

```diff
@@ -606,6 +613,11 @@ bool AP_Landing::is_throttle_suppressed(void) const
     }
 }
 
+//defaults to false, but _options bit zero enables it.
+bool AP_Landing::use_thr_min_during_flare(void){ 
+    return OptionsMask::ON_LANDING_FLARE_USE_THR_MIN & _options;
```

Comment:
when checking a bit to make a booean, use 
  return (bitmask & XXX) != 0;

Reply:
fixed in latest push